src/main/java/states/ReadState.java

```diff
@@ -25,6 +24,11 @@
 	private SocketAddress serverAddress;
 	private String filename;
 
+    private static final Logger LOG = new Logger("FTPClient - Read");
+    private static final String READ_OPERATION = "read";
```

Comment:
Is this used anywhere?

Reply:
It is not...not sure why that appeared under my changes. Will delete.